pkgs/applications/virtualization/seabios/default.nix

```diff
@@ -1,4 +1,11 @@
-{ lib, stdenv, fetchgit, acpica-tools, python3 }:
+{ lib, stdenv, fetchgit, acpica-tools, python3
```

Comment:
(the changes to this file might still be kinda nice to have, but no big deal anymore I guess)